src/charmed_kubeflow_chisme/components/charm_reconciler.py

```diff
@@ -207,8 +214,18 @@ def _update_charm_status(self):
         """Computes Component statuses, updating the attached Charm's with the aggregate status.
 
         Also logs the full status details to the charm logs.
+
+        By default, if this CharmReconciler has no components then it is active.
         """
         statuses = self._get_component_statuses()
+        print("##########################")
+        print(len(self._component_graph))
+        print("##########################")
```

Comment:
Do we need it? When is that useful?

Reply:
@ca-scribner let me know in our live discussion that this is a left over from debugging. I'm removing it thus.